community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/state/storeview/DynamicIndexStoreView.java

```diff
@@ -69,13 +83,43 @@ public DynamicIndexStoreView( LabelScanStore labelScanStore, LockService locks,
     {
         if ( forceStoreScan || !USE_LABEL_INDEX_FOR_SCHEMA_INDEX_POPULATION || useAllNodeStoreScan( labelIds ) )
```

Comment:
since now we will always use native label index i think we should o use it always to determine affected nodes or at least change threshold since we do not have the same problem here as we had with lucene index that we need to reopen before quite often to observe latest updates